<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelInspectorWidget</class>
 <widget class="QWidget" name="ModelInspectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>146</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="showAllModels">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Show All Points</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="selectedModelsOnly">
       <property name="text">
        <string>Show Selected Points Only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="modelTable"/>
   </item>
   <item>
    <widget class="QLabel" name="statusLine">
     <property name="text">
      <string>No of  Points: 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
